integration/apps/arithmetic_intensity/arithmetic_intensity.py

```diff
@@ -33,83 +33,82 @@
     """
+    parser.epilog =  "Following are the additional/optional arguments passed to the app:\n\n" + read_app_help("sse")
+    parser.formatter_class = argparse.RawDescriptionHelpFormatter
     parser.add_argument('--run-type', dest='run_type',
-        choices=['sse', 'avx2', 'avx512'], default='sse',
-        help='Choose a vectorization type for the run.')
-    parser.add_argument('--slowdown', type=float, default=1,
-        help='When imbalance is present, this specifies the amount of work for slow ranks' +
-             ' to perform, as a factor of the amount of work the fast ranks perform.')
-    parser.add_argument('--base-internal-iterations', type=int, default=1,
-        help='How many iterations to perform in the inner loop, for the fast set of ranks ' +
-             '(all ranks if there is no imbalance).')
-    parser.add_argument('--slow-ranks', type=int, default=0,
-        help='The number of ranks to run with extra work for an imbalanced load.')
-    parser.add_argument('--floats', type=int, default=67108864,
-        help='The number of floating-point numbers per rank in the problem array.')
-    parser.add_argument('-v', '--verbose',
-        help='Run in verbose mode.')
-    parser.add_argument('-s', '--single-precision',
-        help='Run in single-precision mode.')
-    parser.add_argument('-l', '--list', action='store_true',
-        help='List the available arithmetic intensity levels for use with the --benchmarks option.')
-    parser.add_argument('-i', '--iterations', type=int, default=5,
-        help='The number of times to run each phase of the benchmark.')
-    parser.add_argument('-b', '--benchmarks', nargs='+',
-        type=float, choices=[0, 0.25, 0.5, 1, 2, 4, 8, 16, 32],
-        help='List of benchmark intensity variants to run (all are run if this option is not' +
-             ' specified).')
-    parser.add_argument('--start-time', type=int,
-        help='Time at which the benchmark will start, in seconds since the system clock\'s epoch.' +
-             ' Start immediately by default, or if the provided time is in the past.')
+                        choices=['sse', 'avx2', 'avx512'], default='sse',
+                        help='Choose a vectorization type for the run (default: sse).')
     parser.add_argument('--ranks-per-node', dest='ranks_per_node',
                         action='store', type=int,
                         help='Number of physical cores to reserve for the app. ' +
                              'If not defined, all nodes but one will be reserved (leaving one ' +
                              'node for GEOPM).')
     parser.add_argument('--distribute-slow-ranks', action='store_true',
                         help='Distribute slow ranks across nodes and packages. Otherwise, slow '
-                             'ranks are assigned to fill nodes and packages.')
+                             'ranks are assigned to fill nodes and packages (default: True).')
```

Comment:
Isn't the default False for store_true actions?

Reply:
You are absolutely right. That was a copy/paste artifact :P